rust/lancedb/src/remote/client.rs

```diff
@@ -59,22 +224,47 @@ impl RestfulLanceDbClient<Sender> {
             });
         }
         let db_name = parsed_url.host_str().unwrap();
+
+        // Get the timeouts
+        let connect_timeout = Self::get_timeout(
+            client_config.timeout_config.connect_timeout,
+            "LANCE_CLIENT_CONNECT_TIMEOUT",
+            Duration::from_secs(120),
+        )?;
+        let read_timeout = Self::get_timeout(
+            client_config.timeout_config.read_timeout,
+            "LANCE_CLIENT_READ_TIMEOUT",
+            Duration::from_secs(300),
+        )?;
+        let pool_idle_timeout = Self::get_timeout(
+            client_config.timeout_config.pool_idle_timeout,
+            // Though it's confusing with the connect_timeout name, this is the
+            // legacy name for this in the Python sync client. So we keep as-is.
```

Comment:
Ah sorry, yeah I guess this is confusing. I guess at the time, the naming was thinking of this from the TCP connection perspective, vs. the client's pool perspective. E.g. this was the time a TCP connection would be idle until it is closed.